website/urls.py

```diff
@@ -12,6 +13,9 @@
     # Cite Page for publications
     url(r'^cite/$', views.cite, name='cite'),
 
+    # News Post display page
+    url(r'^news/(?P<news_id>.*?)/$', views.news_page, name='news_page'),
```

Comment:
Is this reachable from somewhere? No matter where I click I can't find what should lead here...